specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/preview/2015-06-15/Network.json

```diff
@@ -75,6 +75,102 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.Network.Admin/locations": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the list of supported locations.": {
+            "$ref": "./examples/Operations/List.json"
+          }
+        },
+        "description": "Returns the list of supported locations",
+        "tags": [
+          "Network"
+        ],
+        "operationId": "Locations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LocationsList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Network.Admin/locations/{location}/operationResults": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the list of operation results for a location": {
+            "$ref": "./examples/Locations/List.json"
+          }
+        },
+        "description": "Returns the list of operation results for a location",
+        "tags": [
+          "Network"
+        ],
+        "operationId": "LocationsOperationResults_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OperationResultList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Network.Admin/locations/{location}/operations": {
+      "get": {
+        "x-ms-examples": {
+          "Returns the list of support REST operations.": {
+            "$ref": "./examples/Locations/List.json"
```

Comment:
This seems not a matching example json.

Reply:
I believe the example should be valid. Per below, the location operation isn't really supported OnPrem. This operation is here to keep consistency with public azure.

Reply:
What I mean is that this /{location}/**operations** API refer to /**Locations**/List.json, while above /**locations** API refer to /**Operations**/List.json. Seems reversed to me.

Anyway, not an important issue here.